ci-operator/config/operator-framework/operator-sdk/operator-framework-operator-sdk-master.yaml

```diff
@@ -13,6 +18,42 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
+images:
+- from: root
+  dockerfile_path: ci/dockerfiles/builder.Dockerfile
+  to: osdk-builder
+- from: ansible-runner
+  dockerfile_path: ci/dockerfiles/ansible.Dockerfile
+  inputs:
+    osdk-builder:
+      as:
+      - osdk-builder
+      paths: null
+  to: ansible-operator
```

Comment:
We will need to remove the 4.2 jobs for the OCP-release-sdk after this is merged so they don't overwrite each other.